src/py/gee/utils.py

```diff
@@ -639,17 +639,70 @@ def myImageMapper(img):
 def getStatistics(extent):
     extentGeom = ee.Geometry.Polygon(extent)
     elev = ee.Image('USGS/GTOPO30')
-    minmaxElev = elev.reduceRegion(
-        ee.Reducer.minMax(), extentGeom, 1000, maxPixels=500000000)
-    minElev = minmaxElev.get('elevation_min').getInfo()
-    maxElev = minmaxElev.get('elevation_max').getInfo()
-    ciesinPopGrid = ee.Image('CIESIN/GPWv4/population-count/2020')
-    popDict = ciesinPopGrid.reduceRegion(
-        ee.Reducer.sum(), extentGeom, maxPixels=500000000)
-    pop = popDict.get('population-count').getInfo()
-    pop = int(pop)
-    return {
-        'minElev': minElev,
-        'maxElev': maxElev,
-        'pop': pop
-    }
+    ciesinPopGrid = ciesinPopGrid =ee.ImageCollection("CIESIN/GPWv411/GPW_Population_Count").sort('system:time_start',False).first()

```

Comment:
I think `cieisinPopGrid` is a typo. Can you rename this to `citiesInPopGrid` (or `citiesinPopGrid`, I'm not sure the proper Python naming conventions)? 

Also, what is the `=ee.ImageCollection` syntax? I haven't seen that before.

Reply:
Oops definitely a typo with the double variable assigning. I had to look it up but I think it is supposed to be "ciesin" - it is an acronym for "Center for International Earth Science Information Network." But having that level of distinction is kind of pointless ha. I'll rename it to `population`.   

The `ee.ImageCollection` is one of [GEE's data structures](https://developers.google.com/earth-engine/guides/ic_creating). They're used to store raster's which you can filter, map, and reduce against. Similarly there is an `ee.FeatureCollection` which functions in the same way but for vector data.

Reply:
Ah, gotcha - that makes sense. I thought perhaps there was some special GEE Python syntax that I hadn't come across yet since that didn't line didn't throw a (noticeable) error.

Thanks for the link to the GEE docs - GEE is something I don't have a lot of experience with but would love to learn more about.

Yeah I've normally seen Python code using `snake_case`, though it seems you're right in that GEE uses `camelCase` in their Python code to match their JavaScript API (`camelCase` is the standard in JavaScript).

Reply:
No problem on the documentation link - I am pretty much living in there haha. We have tons of GEE resources I can send your way if you want, or if you ever have a question I'm happy to chat or hack on it with you.

Reply:
I'd love to take a look at any GEE resources you have, thank you! Feel free to DM/email them to me :student: